Cell p
---
Cell[] black
Cell[] white
